### PR TITLE
Support Docker forcePullImage for v0

### PR DESCRIPTION
**Pull request by kensipe**
Support Docker forcePullImage for v0

Summary:
https://jira.mesosphere.com/browse/METRONOME-207

JIRA issues:  METRONOME-207

### CODE REVIEW

File: api/src/main/resources/public/api/v0/schema/jobspec_v0.schema.json [hunk 1]
```diff
@@ -86,6 +86,10 @@
             "image": {
               "type": "string",
               "documentation": "The docker repository image name."
+            },
+            "forcePullImage": {
+              "type": "boolean",
+              "documentation": "The containerizer will pull the image from the registry, even if the image is already downloaded on the agent."
```
Comment (ichernetsky) [inline api/src/main/resources/public/api/v0/schema/jobspec_v0.schema.json]: `on the agent` -> ` on the agent node`?

File: api/src/test/scala/dcos/metronome/api/v1/controllers/JobSpecControllerTest.scala [hunk 2]
```diff
@@ -45,6 +45,30 @@ class JobSpecControllerTest extends PlaySpec with OneAppPerTestWithComponents[Mo
       contentType(response) mustBe Some("application/json")
     }
 
+    "creates job when sending docker spec with forcePullImage property against v0" in {
+      Given("Job spec with forcePullImage")
+      val specJson = "{\"id\":\"spec1\",\"run\":{\"cpus\":1,\"mem\":128,\"disk\":0,\"docker\":{\"image\":\"image\", \"forcePullImage\": true}}}"
```
Comment (ichernetsky) [inline api/src/test/scala/dcos/metronome/api/v1/controllers/JobSpecControllerTest.scala]: Maybe it is worth using triple quotes here? Like
```
@ """{"foo":"bar"}""" 
res0: String = "{\"foo\":\"bar\"}"
```

Comment (kensipe) [reply api/src/test/scala/dcos/metronome/api/v1/controllers/JobSpecControllerTest.scala]: I was stay consistent... and truth be known, I copied a previous test and modified for this purpose... but yeah... this should be the way

**Comment by timcharper** [comment]
How does this signal get ultimately passed to Mesos?

**Comment by timcharper** [comment]
Was it already done but v0 api just needed JSON schema change?

**Comment by kensipe** [comment]
most of the work was in #158 however this required forcepullimage which was fixed by #167 however all of this work was on the `/v1` endpoint and not `/v0`.   It turns out the UI uses the v0 end point.

**Comment by kensipe** [comment]
rebased

**Pull request merged**